src/schema/2.0/w3c.test.ts

```diff
@@ -0,0 +1,76 @@
+import { issueDocument } from "../../index";
+import sampleDoc from "./sample-document.json";
+import { mapToW3cVc } from "./w3c";
+
+describe("it should be correct", () => {
+  it("should really be correct", () => {
+    const document = issueDocument(sampleDoc);
```

Comment:
Will it make sense to use a fixture here so that we can test the deterministic conversion between one OA Document to one VC document?

Reply:
- translation for me => use an envelope

Reply:
done